src/common/datatypes/HostAddr.h

```diff
@@ -64,6 +64,10 @@ struct HostAddr {
     ha.port = std::stoi(str.substr(pos + 1));
     return ha;
   }
+
+  static HostAddr nullAddr() {
```

Comment:
same with default constructor

Reply:
Just wan't to give it a readable name~

Reply:
confused with nullptr

Reply:
I'll it's better than using` HostAddr("", 0)` directly? Of course, the name could be improved.

Reply:
Maybe return static object reference or Constant value is better?